docs/rules/trials.md

```diff
@@ -0,0 +1,80 @@
+
+##Case Types
+
+Usually a trial will happen to address one of two types of cases:
+
+- Civil Cases: A civil case is one brought by individual against another in order to resolve a dispute not covered by [[SpaceLaw | Space Law]]. For example, the bartender being sued for banning a patron from the bar.
+
+- Criminal Case: A criminal case is one brought from the station authority such as Security, or Command for violations of Space Law. These cases are usually about severe incidents involving High Crimes. For example security may bring charges against an engineer for sabotaging the Anti Matter Engine.
+
+While it may seem there is not much of a distinction, the biggest difference between the two are that Civil Cases usually do not carry any criminal punishment for the defendant, and are usually held to resolve a disagreement between crew members. In contrast a criminal case usually deals with serious crimes against the station which may lead to punishments such as permanent confinement or even execution.
+
+###Required Personnel
+
+* **Judge**: A Judge must be appointed. This may be the Captain, the Head of Personnel, or any authorized individual. The role of the Judge is to preside over the proceedings, resolve any disputes arising from them, and pass sentencing. Additionally during a bench trial it is also the Judge's responsibility to decide on a verdict for the defendant.
+* **Prosecutor**: Prosecutor must be appointed in Criminal Cases. This can either be the warden, a security officer familiar with the case, or any other authorized individual. The prosecutor's role is **not** to get a guilty verdict at all costs, but to uncover the truth and act in the best interests of justice. In Civil case, this role is fulfilled by an attorney of the opposing side who will prosecute the defendant on the plaintiff's<ref>The individual suing in a Civil Case</ref> behalf.
+* **Defense Attorney**: The defendant in a criminal trial is entitled to an Attorney, and in Civil Cases both parties may acquire one. This may be a lawyer, or any other competent individual. The attorney's role is to zealously represent their client, and attain the most favorable verdict possible.
+* **Jury**: A Jury may be formed in order to decide the verdict. The Jury may be composed of any amount of members of the crew, and the number of jurors is at the discretion of the Judge. Additionally, a bench trial may be conducted instead where instead of a Jury the verdict is decided by judge, or a panel of Judges. Jurors must be impartial, and unrelated to the case.
+* **Bailiff**: A Bailiff must be appointed by the judge. This may be the Head of security, the warden, or any authorized individual. The Bailiff's role is to ensure the security of the trial, and command security personnel in the courtroom.
```

Comment:
Update this list to also use the `- Example text` list syntax rather than the asterisk. It's confusing to use both and the dash is used in every other document as well.